app/views/shared/_nav.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="menu">
+  <p class="nav-color"><%= link_to "All Users", users_path %></p>
+  <p class="nav-color"><%= link_to "All Posts", posts_path %></p>
+  <p class="nav-color"><%= link_to "Create Post", new_user_post_path(active_user.id) %></p>
```

Comment:
- [ ] For now, I am unable to run the project because there is no current user. Please make sure that you make some user in the seed data and give a command in the README as well. `rails db:seed` to make sure we have a current user which will perform making posts, comments, and likes.

I'm getting this error while running your project

![Screenshot from 2021-12-10 08-31-37](https://user-images.githubusercontent.com/22434660/145510386-20d9b100-4d98-4b44-a32a-71bd3d2eb583.png)